src/main/java/com/amplitude/Response.java

```diff
@@ -22,8 +33,9 @@ protected static Response populateResponse(JSONObject json) {
       res.successBody.put("payloadSizeBytes", json.getInt("payload_size_bytes"));
       res.successBody.put("serverUploadTime", json.getLong("server_upload_time"));
     } else if (status == Status.INVALID) {
-      res.invalidRequestBody = new JSONObject();
       res.error = Utils.getStringValueWithKey(json, "error");
+      if (hasInvalidAPIKey(res.error)) throw new AmplitudeInvalidAPIKeyException(res.error);
```

Comment:
nit: let's make it two lines

Reply:
Will not fix this since it's the result after running google java format.